dev_guide/quota.adoc

```diff
@@ -11,59 +11,99 @@ toc::[]
 
 == Overview
 
-A *ResourceQuota* object enumerates hard resource usage limits per project. It
-limits the total number of a particular type of object that may be created in
-a project, and the total amount of
-link:compute_resources.html[compute resources]
+A *ResourceQuota* defines constraints that limit aggregrate resource consumption
+per project.  It can limit the quantity of objects that can be created in a project
+by type, as well as the total amount of link:compute_resources.html[compute resources]
 that may be consumed by resources in that project.
 
-== Usage Limits
+== Resources managed by quota
 
-The following describes the set of limits that may be enforced by a
-*ResourceQuota*.
+The following describes the set of resources that may be managed by a *ResourceQuota*.
 
-.Usage limits
+.Resources managed by quota
 [cols="3a,8a",options="header"]
 |===
 
 |Resource Name |Description
 
 |`cpu`
-|Total requested cpu usage across all containers
+|Across all pods in a non-terminal state, the sum of cpu requests cannot exceed this value.
 
 |`memory`
-|Total requested memory usage across all containers
+|Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value.
 
-|`openshift.io/imagesize` [[imagesize]]<<zero,*>>
-|Maximum size of a single image stored in an internal registry
+|`requests.cpu`
+|Across all pods in a non-terminal state, the sum of cpu requests cannot exceed this value.
 
-|`openshift.io/imagestreamsize` [[imagestreamsize]]<<zero,*>>
-|Maximum size of image stream's images stored in an internal registry
```

Comment:
There are some references to these resources in `install_config/install/docker_registry.adoc`, but I can deal with them in the follow-up.